package/MDAnalysis/analysis/align.py

```diff
@@ -796,13 +797,8 @@ def __init__(self, mobile, reference=None, select='all', filename=None,
           already a :class:`MemoryReader` then it is *always* treated as if
           ``in_memory`` had been set to ``True``.
 
-        .. versionadded:: 1.0.0
-
-        .. versionchanged:: 1.0.0
```

Comment:
oops, that one's on me I believe.

Reply:
No biggie — I rather have more information than less.

Thanks for reviewing so quickly.... and the other one — you beat me to asking for your review.